airflow/decorators/base.py

```diff
@@ -305,13 +311,21 @@ def expand(self, **map_kwargs: "Mappable") -> XComArg:
         prevent_duplicates(self.kwargs, map_kwargs, fail_reason="mapping already partial")
         ensure_xcomarg_return_value(map_kwargs)
 
-        partial_kwargs = self.kwargs.copy()
+        task_kwargs = self.kwargs.copy()
+        dag = task_kwargs.pop("dag", None) or DagContext.get_current_dag()
+        task_group = task_kwargs.pop("task_group", None) or TaskGroupContext.get_current_task_group(dag)
+
+        partial_kwargs, default_params = get_merged_defaults(
+            dag=dag,
+            task_group=task_group,
+            task_params=task_kwargs.pop("params", None),
+            task_default_args=task_kwargs.pop("default_args", None),
```

Comment:
`default_args` isn't a valid argument to Operators is it?

Reply:
It actually is, there’s a `default_args` argument on BaseOperator.

Reply:
Oh well that's just weird. I wonder why it's there.